draft-ietf-rtcweb-jsep.xml

```diff
@@ -1885,15 +1885,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           session-level attributes. The process here is identical to that
           indicated in the Initial Offers section above.</t>
 
-	  <t>The next step is to generate lip sync groups as defined
-	  in <xref target="RFC5888"/>, Section 7.  For each
-	  MediaStream with more than one MediaStreamTrack, a group
-	  of type "LS" MUST be added that contains the mid values for
-	  each MediaStreamTrack in that MediaStream. In some cases
-	  this may result in adding a mid to a given LS group that
-	  was not in that LS group in the associated offer. Although
-	  this is not allowed by <xref target="RFC5888"/>, it is
-	  allowed when implementing this specification. </t>
+	  <t>The next step is to generate session-level lip sync
+	  groups as defined in <xref target="RFC5888"/>, Section
+	  7. All Tracks in the same MediaStream need to be placed in
+	  the same LS group.  RFC5888 only allows the groups in the
+	  answer to be a subset of what was in the offer. If the
+	  answer wishes to have two things synchronized that were not
+	  in the same LS group in the in the offer, another offer
```

Comment:
I don't think this is sufficient guidance; consider the case where the caller does not want to sync audio/video, but the callee does. The caller offers an empty LS group, and because 5888, the callee can only reply with an empty LS group. But when the callee reoffers a LS group with audio and video, the caller will reject that LS group, since it doesn't match its expectations.

I think we need to simply say that such a case doesn't work right now, and is left as a problem for future study. (In reality I expect all implementations to just implement declarative behavior.)